README.md

```diff
@@ -236,6 +236,6 @@ correctness as part of our continuous integration system.
 [install-ubuntu]: https://plaidml.github.io/plaidml/docs/install#ubuntu
 [install-macos]: https://plaidml.github.io/plaidml/docs/install#macos
 [install-windows]: https://plaidml.github.io/plaidml/docs/install#windows
-[EDSL]: docs/docs/edsl.md
+[EDSL]: https://plaidml.github.io/plaidml/docs/edsl.html
```

Comment:
I think we should elide the `.html`

Reply:
cool, didn't know that was a thing! I just copied what the navbar directed me to